easybuild/easyconfigs/c/CSBLAST/CSBLAST-2.2.4-GCCcore-11.3.0.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'CSBLAST'
+version = '2.2.4'
+
+homepage = 'https://github.com/soedinglab/csblast/'
+description = """Context-specific extension of BLAST that significantly improves sensitivity and alignment quality."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+github_account = 'soedinglab'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['76848da4d45a618ae903cafc00ff6387e7decb17b839aca83d9a9438537edf0d']
+
+builddependencies = [
+    ('binutils', '2.38'),
+]
+
+dependencies = [
+    ('sparsehash', '2.0.4'),
+]
+
+start_dir = 'src'
+
+build_cmd_targets = 'csblast csbuild'
+
+buildopts = '"FLAGS=-fpermissive  -std=c++11 "'
```

is that double quotation here really correct? I would expect
```suggestion
buildopts = 'FLAGS=-fpermissive  -std=c++11 '
```

---

I think I set it at the wrong place. Without any double quotes I get:
```
ERROR EasyBuild crashed with an error (at easybuild/tools/build_log.py:111 in caller_info): cmd " make csblast csbuild -j 8 FLAGS=-fpermissive -s
td=c++11 " exited with exit code 2 and output:
make: invalid option -- '='
make: invalid option -- 'c'
make: invalid option -- '+'
make: invalid option -- '+'
make: invalid option -- '1'
make: invalid option -- '1'
Usage: make [options] [target] ...
```
However, it works like that:
```
buildopts = 'FLAGS="-fpermissive -std=c++11" '
```